stac_fastapi/pgstac/setup.py

```diff
@@ -0,0 +1,65 @@
+"""arturo-stac-api."""
+import os
+from imp import load_source
+
+from setuptools import find_namespace_packages, setup
+
+with open("README.md") as f:
+    desc = f.read()
+
+# Get version from stac-fastapi-api
+__version__ = load_source(
+    "stac_fastapi.pgstac.version",
+    os.path.join(os.path.dirname(__file__), "stac_fastapi/pgstac/version.py"),
+).__version__  # type:ignore
+
+install_requires = [
+    "fastapi",
+    "attrs",
+    "orjson",
+    "pydantic[dotenv]",
+    "stac_pydantic==1.3.8",
+    "stac-fastapi.types",
+    "stac-fastapi.api",
+    "stac-fastapi.extensions",
+    "asyncpg",
+    "buildpg",
+    "shapely",
+    "mangum",
```

Comment:
Just curious: is `mangum` only temporary? Seems like it would be an optional dependency for users who use Lambda.

Reply:
moved to a try block and moved install to awslambda extras